tests/migrations/test_migrations.py

```diff
@@ -397,3 +397,13 @@ def test_migrations_to_ba8234fe9eb5(alembic_runner: MigrationContext, alembic_en
     inspector = sqlalchemy.inspect(alembic_engine)
 
     assert "phone_ext" in set([c["name"] for c in inspector.get_columns("contact_info")])
+
+
+def test_migrations_to_f4091e4ce218(alembic_runner: MigrationContext, alembic_engine: Engine):
+    alembic_runner.migrate_up_to("f4091e4ce218")
+
+    inspector = sqlalchemy.inspect(alembic_engine)
+
+    assert "user_name" in set([c["name"] for c in inspector.get_columns("user_action")])
+    assert "user_email" in set([c["name"] for c in inspector.get_columns("user_action")])
+    assert "user_id" in set([c["name"] for c in inspector.get_columns("user_action")])
```

Comment:
since the change is to the data type, and not add / removing the columns, checking the data type has length restrictions seem more appropriate, see if the inspector is able to give back data type information with the length info.

Reply:
I checked and I think there was no way to get column type, but I will check again to make sure. I might be wrong.

Reply:
should be able to use `hasattr(column["type"], "length")` to see if the column has a length limit, then `column["type"].length` to get the value of that limit.

Reply:
Thanks Le